packages/components/src/components/post-popovercontainer/post-popovercontainer.scss

```diff
@@ -35,6 +35,7 @@
     background-color: var(--post-current-palette-bg);
     border: 2px solid transparent; // Keeping the default border for HCM
     border-radius: commons.$border-radius;
+    max-width: var(--post-popover-max-width);
```

Comment:
The post-popover.scss also has a max-width defined. Setting the variable on the popovercontainer has some side effects, namely that it's available for other components using the popovercontainer like tooltip, menu, etc. I think this specific variable should be set on the popover component specifically.